tests/e2e/index.spec.ts

```diff
@@ -179,25 +189,6 @@ test.describe("Index page", () => {
         await expect(await summary).toHaveAttribute("feature-count", "1915");
     });
 
-    test("admin level toggle works", async ({ page }) => {
-        const adminToggle = await page.locator("#admin-toggle");
-
-        await expect(adminToggle).toHaveCount(0);
-
-        const allRegions = await page.locator(GEOJSON_SELECTOR);
-        const firstRegion = await getNthRegion(page, 1);
-        await firstRegion.click();
-        await page.waitForURL(/dengue\/may24\/FOI\/AGO\/admin2/i);
-
-        await expect(await allRegions).toHaveCount(2060);
-
-        await expect(adminToggle).toHaveCount(1);
-        await adminToggle.getByRole("button", { name: "Admin 1" }).click();
-        await page.waitForURL(/dengue\/may24\/FOI\/AGO\/admin1/i);
-
-        await expect(await allRegions).toHaveCount(1915);
-    });
-
```

Comment:
Goodbye test. Why is it being deleted btw?

Reply:
apologies, i was going to add this back in but forgot, it is added back in now!